src/wp-includes/post.php

```diff
@@ -8018,3 +8019,38 @@ function use_block_editor_for_post_type( $post_type ) {
 	 */
 	return apply_filters( 'use_block_editor_for_post_type', true, $post_type );
 }
+
+/**
+ * Adds sync_status meta field to the wp_block post type so an unsynced option can be added.
+ *
+ * @since 6.3.0
+ *
+ * @see https://github.com/WordPress/gutenberg/pull/51144
```

Comment:
```suggestion
 * @link https://github.com/WordPress/gutenberg/pull/51144
```

Nit but I think `@see` is intended for internal function and hook links.

Reply:
Done.